doc/integrator/intranet.rst

```diff
@@ -33,11 +33,12 @@ This page lists the changes that must be applied to add such a functionality.
            if 'anonymous' in request.params:
                return None
 
-           get_user_from_request = _create_get_user_from_request(request.registry.settings)
+           get_user_from_request = create_get_user_from_request(request.registry.settings)
            user = get_user_from_request(request)
 
            if user is None and request.environ.get('intranet', 0) == '1':
-               from c2cgeoportal_commons.models import DBSession, Role
+               from c2cgeoportal_commons.models import DBSession
+               from c2cgeoportal_commons.models import Role
```

Comment:
Why this change?

Reply:
I thought this was our preferred style... but actually, I see we have both styles so I will undo this change.